source/renderer/app/index.js

```diff
@@ -27,11 +27,10 @@ configure({
 addLocaleData([...en, ...de, ...hr, ...ja]);
 
 const environment = global.environment;
-const { NODE_ENV, NETWORK } = environment;
-const isTest = NODE_ENV === 'test';
+const { isTest, network } = environment;
 
 const initializeDaedalus = () => {
-  const api = setupApi(isTest, String(NETWORK));
+  const api = setupApi(isTest, String(network));
```

Comment:
Great catch @MarcusHurney 🚀

Reply:
@nikolaglumac 
I initially had the same ipc message to re-render the application menu in place for the Select Language screen but I didn't know if it was needed. I can add that in.